### PR TITLE
Adds an option to not persist autogroups

### PR DESCRIPTION
**Pull request by L-e-x-o-n**
Link to issue #702
Works, but it will start off as disabled (don't persist) which is opposite of current behavior, does anyone know how I can make an option start as enabled by default?

### CODE REVIEW

File: language/en.lua [hunk 1]
```diff
@@ -541,6 +541,8 @@ return {
 			unitreclaimer_descr = 'Reclaim units in an area. Hover over a unit and drag an area-reclaim circle',
 			autogroup_immediate = 'Autogroup immediate mode',
 			autogroup_immediate_descr = 'Units built/resurrected/received are added to autogroups immediately,\ninstead when they get to be idle.\n\n(add units to autogroup with ALT+number)',
+			autogroup_persist = 'Autogroup persist mode',
+			autogroup_persist_descr = 'Autogroups persist in new games.\nIf this is enabled units which were added to an autogroup in previous games\n will be added to the same group in this game.',
```
Comment (badosu) [inline language/en.lua:545]: @Ruwetuin Can you review this text? Does it look good for you?

**Comment by badosu** [comment]
LGTM :+1: 

You can add "Closes #702" or "Fixes #702" in the body of the commit or description message of the PR and it will automatically close that issue.

I pinged @Ruwetuin to see if he approves the text.

I assume you tested toggling persist off and on again right?

As for default persist disabled that sounds good to me, I'd get feedback from others though.

**Comment by L-e-x-o-n** [comment]
Nice, thanks. 

> LGTM 👍
> 
> You can add "Closes #702" or "Fixes #702" in the body of the commit or description message of the PR and it will automatically close that issue.
> 
> I pinged @Ruwetuin to see if he approves the text.
> 
> I assume you tested toggling persist off and on again right?
> 
> As for default persist disabled that sounds good to me, I'd get feedback from others though.

I was trying this https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue on other PRs but it doesn't work always, maybe it need to be in the comment and not title?

I tested it in singleplayer and it worked for me.

**Comment by badosu** [comment]
You can just add the comment to the description of the PR, something in the format of "Fixes #702"

**Pull request merged**